src/components/Aqidata.jsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import millify from 'millify';
+import Header from './Header';
+import { displayWeatherData } from '../redux/Aqidata/aqidataSlice';
+
+const Aqiweather = () => {
+  const location = useLocation();
+  const { city } = location.state;
```

Comment:
* I get errors on my console when i try to run your application , kindly work on it 
![image](https://github.com/Kanyorok/air-pollution-aqi/assets/52497246/ab540941-0381-4435-af4a-51f561385185)